generators/app/templates/src/app/components/Button/index.js

```diff
@@ -7,28 +7,13 @@ import styles from './styles';
 
 const variants = ['small', 'big', 'disabled'];
 
-function Button({
-  onPress,
-  children,
-  link,
-  type,
-  disabled,
-  containerSetter,
-  ...props
-}) {
-  const style = variants
-    .filter(variant => props[variant])
-    .map(variant => styles[variant]);
-  const textStyle = variants
-    .filter(variant => props[variant])
-    .map(variant => styles[`text${variant}`]);
+function Button({ onPress, children, link, type, disabled, containerSetter, ...props }) {
```

Comment:
`onClick` ? So we make this API parallel

Reply:
This'll be erased in a future PR